hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/ha/SequenceIdGenerator.java

```diff
@@ -69,7 +69,12 @@ public class SequenceIdGenerator {
   public static final String LOCAL_ID = "localId";
   public static final String DEL_TXN_ID = "delTxnId";
   public static final String CONTAINER_ID = "containerId";
+
+  // TODO: It can be deleted after several Ozone releases 
+  @Deprecated
   public static final String ROOT_CERTIFICATE_ID = "rootCertificateId";
+  // certificate ID for all services
```

Comment:
This comment doesn't give too much new information this way, does this mean that instead of the `ROOT_CERTIFICATE_ID` this is now used for all services? You could mention it perhaps.